cabal-install/src/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -862,8 +862,10 @@ printPlan verbosity
           ProjectBaseContext {
             buildSettings = BuildTimeSettings{buildSettingDryRun},
             projectConfig = ProjectConfig {
+              projectConfigAllPackages =
+                  PackageConfig {packageConfigOptimization = globalOpt},
```

Comment:
It is a big function so i would be more verbose
```suggestion
                  PackageConfig {packageConfigOptimization = globalOptimization},
```
or something alike

Reply:
Done, thank you. (I'm not committing it straight from the GH suggestion because it needs renaming it at the use sight.)

Reply:
👍 dont worry at all, i use them cause they are usually clearer when proposin changes